### PR TITLE
Accounts v1/pt2

### PR DESCRIPTION
**Pull request by acollette**
-Have to refactor inputs of accountManagementAction to get a better solution for stackToDeep

### CODE REVIEW

File: src/accounts/AccountV1.sol [hunk 1]
```diff
@@ -550,34 +550,37 @@ contract AccountV1 is AccountStorageV1, IAccount {
     {
         if (!IRegistry(registry).isActionAllowed(actionHandler)) revert AccountErrors.ActionNotAllowed();
 
+        address actionHandler_ = actionHandler;
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:553]: not storage so no need to cache it

Comment (acollette) [reply src/accounts/AccountV1.sol:553]: It's to avoid stack too deep. I'm trying to find a better solution but have some issues

Comment (acollette) [reply src/accounts/AccountV1.sol:553]: I'm trying to switch inputs and have` actionHandler` as last input, but gives weird errors when I adapt rest of the functions

Comment (acollette) [reply src/accounts/AccountV1.sol]: ok working now

**Comment by Thomas-Smets** [comment]
I actually meant it opposite in my comment: that we pass the actiondata not coded to doAccountManagement Action, and that for the actionHandler we just give a single bytes object.
That has to be decoded in the actionhandler.

**Comment by Thomas-Smets** [comment]
Since if the actionHandler becomes permissionless, there is no reason they must use as input:
ActionData memory actionHandlerData,
address[] memory to,
bytes[] memory data

File: src/accounts/AccountV1.sol [hunk 2]
```diff
@@ -382,6 +382,17 @@ contract AccountV1 is AccountStorageV1, IAccount {
...
     }
 
+    /**
+     * @notice Checks if the Account is healthy and still has free margin.
+     * @return success Boolean indicating if there is sufficient margin to back the debt of the Account.
+     * @return creditor_ The contract address of the Creditor.
+     * @return accountVersion_ The Account version.
+     * @dev An Account is healthy if the collateral value is bigger than or equal to the used margin.
+     */
+    function isAccountHealthy() external view returns (bool success, address creditor_, uint256 accountVersion_) {
```
Comment (j-vp) [inline src/accounts/AccountV1.sol:392]: where is this function used?

**Comment by Thomas-Smets** [comment]
Merged branch in https://github.com/arcadia-finance/accounts-v2/pull/100

**Pull request closed without merging**